js/modules/sitemap.js

```diff
@@ -0,0 +1,76 @@
+/* global sitemap */
```

Comment:
This will need documentation for the file header. https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#file-headers

Might also help to link to JS chosen somewhere (either file header of function).

Reply:
done